app/components/UI/Ramp/Deposit/Views/BasicInfo/BasicInfo.tsx

```diff
@@ -121,85 +128,111 @@ const BasicInfo = (): JSX.Element => {
     }
   }, [navigation, validateFormData, formData, quote, kycUrl]);
 
+  const handleSubmitEditing = useCallback(
+    (nextRef: React.RefObject<TextInput>) => {
+      nextRef.current?.focus();
+    },
+    [],
+  );
```

Comment:
to avoid creating arrow functions refactor this to a callback that returns the function 
```suggestion
  const handleSubmitEditing = useCallback(
    (nextRef: React.RefObject<TextInput>) => () =>  {
      nextRef.current?.focus();
    },
    [],
  );
```

later in the code
```diff
- onSubmitEditing={() => handleSubmitEditing(lastNameInputRef)}
+ onSubmitEditing={handleSubmitEditing(lastNameInputRef)}
```